src/OnboardingSPA/pages/Steps/DesignHomepageMenu/stylesheet.scss

```diff
@@ -0,0 +1,132 @@
+            justify-content: center;
+
+            &__title-bar {
+                    width: 90%;
+                    height: 15px;
+                    display: flex;
+                    align-items: center;
+                    background-color: #E4E4E4;
+                    justify-content: space-between;
+                    border: 1px solid transparent;
+            
+                    &__browser {
+                        display: flex;
+                        align-items: center;
+                        justify-content: center;
+            
+                        &__dot {
+                            width: 8px;
+                            margin: 3px;
+                            height: 8px;
+                            border-radius: 50%;
+                            display: inline-block;
+                            background-color: #989ea7;
+                        }
+                    }
+            
+                    &--selected {
+                        z-index: 1;
+                        width: 40px;
+                        height: 40px;
+                        display: flex;
+                        margin-right: -15px;
+                        border-radius: 50%;
+                        align-items: center;
+                        justify-content: center;
+                        background-clip: padding-box;
+                        background-color: $main-border-main;
+            
+                        &__path {
+                            fill: $main-color-light;
+                            transform: scale(0.7);
+                        }
+                    }
+            
+                    &--unselected {
+                        display: none;
+                    }
+                }
+            &__live-preview-container {
+				position: relative;
+				width: 90%;
+				align-items: center;
+				border: 1px solid #e3dfdf;
+
+				.live-preview {
+
+					&__container {
+
+						&-custom {
+							width: 100%;
+							overflow: hidden;
+							height: 300px;
+						}
+					}
+				}
+
+            }
+
+            &__live-preview-container {
+                position: relative;
+                width: 90%;
+                align-items: center;
+                border: 1px solid #e3dfdf;
+
+                .live-preview {
+
+                    &__container {
+
+                        &-custom {
+                            width: 100%;
+                            height: 400px;
+                            // Enable Scrolling in Live Preview
+                            overflow-y: scroll;
+
+                            div {
+                                div iframe {
+                                    // Override the Block Preview max-height attribute 
+                                    // to show the complete Preview
+                                    max-height: 6500px !important;
+                                }
+                            }
```

Comment:
I believe we can skip this, I've added max height in the auto block preview as part of my PR